db/migrate/20181218081452_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+
```

Comment:
Do make a migration to add the email column to the user model

Reply:
DONE. :)